src/test/java/StepDefinition/MessengerLoginSD.java

```diff
@@ -0,0 +1,62 @@
+package StepDefinition;
+
+import Helper.Check;
+import Pages.Facebook.LaunchPage;
+import Pages.Facebook.MessengerHomePage;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class MessengerLoginSD {
+
+   MessengerHomePage messengerPage=new MessengerHomePage();
+
+
+@When("^I click on Messenger link$")
+    public void clickMessengerLink()
+{
+        messengerPage.clickMessengerLink();
+    }
+
+    @When("^I Click Log in button on Messenger screen with empty credentials")
+    public void clickMessengerLogin()
+    {messengerPage.clickMessengerLogInButton();
+    }
+
+    @When("^I click on login button$")
+    public void clickLogin()
+    {
+        messengerPage.clickMessengerLink();
+
+    }
+
+    @When("^I verify login error '(.+)' is displayed$")
+    public void verifyLoginError(String errorMsg) {
+        switch (errorMsg) {
+            case "incorrect email or password":
+                // verify incorrect email or password msg is displayed
+                break;
+            case "enter credentials again":
+                // verify enter credentials again msg is displayed
+                break;
+            case "Find your account and log in":
+                // verify your account details msg is displayed
+                break;
+            default:
+                System.out.println("Not implemented for " + errorMsg + " msg");
```

Comment:
You are not doing any Assertion inside any `case`. You need to do Assertion for any verification.